source/data_api.html.md

```diff
@@ -888,14 +888,14 @@ New messages can be created in a [Patient](#patients) Profile. Messages created
 > Example Request
 
 ```shell
-curl -XPOST /v1/app_messages -d '{
+curl -XPOST https://api.welkinhealth.com/v1/app_messages -d '{
   "patient_id": "65ae66fa-d1c0-4b98-bf0a-21cd6090229f",
   "worker_id": "a1fa82d9-19e0-4114-a6d1-6745f8eaeff0",
   "conversation_id": "2e045bdd-0083-4341-bc37-9a81d990da31",
   "direction": "inbound",
   "contents": "Hi Developer, Welcome to Welkin Health.",
   "sent_at": "2018-09-12T01:27:32.045046+00:00"
-}'
+}' -H "Authorization: Bearer <your access token>" -H "Content-Type: application/json"
```

Comment:
I've never used a Content-Type header and I've never had a problem... but no harm in having it